constellations/gemini-admin/src/index.js

```diff
@@ -47,7 +47,7 @@ const SemesterEditMatch = ({ match }) => (
     <SemesterEditPage semesterId={match.params.semesterId} />
 );
 
-const ClassAll = () => <ClassAllPage />;
+const Class = () => <ClassPage />;
```

Comment:
Thanks for refactoring. I think I had a reason before, but then forgot why. Maybe because `class` is a keyword and `Class` looks too similar.